src/NuGet.Clients/NuGet.PackageManagement.UI/Controls/TabItemButton.cs

```diff
@@ -22,4 +22,4 @@ public void Select()
             OnClick();
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
there is no significant change in this file. 
options:
a) keep the change in...end up with cr-lf at end
b) undo the change via "git checkout dev -- <PATHTOFILE>", i believe -- so the file is not changed